rotkehlchen/chain/ethereum/modules/eth2/constants.py

```diff
@@ -11,4 +11,4 @@
 
 BEACONCHAIN_MAX_EPOCH: Final = 9223372036854775807  # This is INT64 MAX. Beacon node API actually returns 18446744073709551615 which is UINT64 MAX # noqa: E501
 
-DEFAULT_VALIDATOR_CHUNK_SIZE: Final = 80
+DEFAULT_VALIDATOR_CHUNK_SIZE: Final = 100  # This is the maximum amount of validators allowed per API request  # noqa: E501
```

Comment:
allowed per API request in both beaconcha.in and when running your own node?

Reply:
running your own node, there's no limit afaics (tested it).